src/renderer/services/app/service.ts

```diff
@@ -23,4 +25,6 @@ const { get$: getNetwork$, set: changeNetwork, get: getCurrentNetworkState } = o
 // So we do need a simple "dirty check" to provide "real" changes of selected network
 const network$ = getNetwork$.pipe(distinctUntilChanged())
 
-export { onlineStatus$, network$, changeNetwork, getCurrentNetworkState }
+const clientNetwork$ = network$.pipe(RxOp.map(getClientNetwork))
```

Comment:
just a question here.
can not we just add `RxOp.map(getClientNetwork)` to the initial `network$` stream ?

Reply:
As discussed in DM: We will do it as soon as we know how to deal the switch `chaosnet` -> `mainnet` in the near future.